### DISCUSSION TITLE
部署后，在install界面卡住

### DISCUSSION THREAD

**Discussion by Clint-chan** [langgenius/dify#4102] [General]
![image](https://github.com/langgenius/dify/assets/62379027/a48c3f20-45bc-4617-9fac-afb551e5fad3)
如图所示，设置管理员账户时，一直卡住

**Comment by Bloodshades** [comment]
遇到了一样的问题，怎么解决的？

**Reply by xunjh11** [reply]
遇到了一样的问题，解决了吗

**Reply by Bloodshades** [reply]
> 遇到了一样的问题，解决了吗

没解决，放弃群晖了。使用了WSL，成功部署。

**Reply by billconan2017** [reply]
我也是卡这个界面

**Reply by kof02guy** [reply]
切换一下语言，就可以了。不知道怎么回事。

**Reply by Mugglepro** [reply]
有没有解决办法呀？

**Reply by kof02guy** [reply]
> 有没有解决办法呀？

切换一下语言

**Reply by Mugglepro** [reply]
感谢解答。但是切换语言不行，我是其他架构进行源码部署，调试了一下是cors问题，前端向后端的请求被拒绝了。

**Reply by hushan8** [reply]
> 感谢解答。但是切换语言不行，我是其他架构进行源码部署，调试了一下是cors问题，前端向后端的请求被拒绝了。

这个怎么解决

**Reply by Mugglepro** [reply]
前端请求后端资源的时候会出现CORS跨域资源请求的问题，需要在docker-compose.yml的后端API environment里面设置
      - CONSOLE_CORS_ALLOW_ORIGINS=*
      - WEB_API_CORS_ALLOW_ORIGINS=*
这两个能对控制台和web api进行配置，允许所有的域名进行访问。

**Reply by Airgods** [reply]
请问具体在哪里加呢？后端部署部分吗？还是docker容器启动的部分

**Reply by muggle-stack** [reply]
在docker里面，docker-compose.yaml配置文件grep或者ctrl+f搜一下CONSOLE_CORS_ALLOW_ORIGINS这个变量，把它设置为*

**Comment by feimadecaogaozhi** [comment]
请问最后时怎么解决的啊，一样时部署后一直卡在install，求大佬帮助

**Reply by stack-muggle** [reply]
还好之前有写文档：前端请求后端资源的时候会出现CORS跨域资源请求的问题，需要在docker-compose.yml的后端API environment里面设置
      - CONSOLE_CORS_ALLOW_ORIGINS=*
      - WEB_API_CORS_ALLOW_ORIGINS=*
这两个能对控制台和web api进行配置，允许所有的域名进行访问。

**Reply by soberyyz** [reply]
已经修改了环境仍然加载不出来

**Reply by bleaaach** [reply]
> 已经修改了环境仍然加载不出来

我也是 兄弟解决了吗

**Comment by kaxipig** [comment]
是啊， 有咩有完整的源码部署启动的 流程分享？  始终是提示账号密码错误，  翻遍了

**Comment by jiaohuix** [comment]
我也遇到了，CONSOLE_CORS_ALLOW_ORIGINS和WEB_API_CORS_ALLOW_ORIGINS默认就是*，没有解决install卡住的问题

**Reply by jiaohuix** [reply]
> 我也遇到了，CONSOLE_CORS_ALLOW_ORIGINS和WEB_API_CORS_ALLOW_ORIGINS默认就是*，没有解决install卡住的问题

解决了，我不小心把 .env的DIFY_PORT=5001改成了5003，而nginx/conf.d/default.conf里面用的是http://api:5001，结果就卡在install界面了

**Comment by Clint-chan** [comment]
统一回复一下，基本上是因为你们防火墙没有放行这个端口
